dashboard/src/components/api/json-rpc/websocket-client.ts

```diff
@@ -126,7 +126,9 @@ export class WebsocketClient implements ICommunicationClient {
    * @param data to be sent
    */
   send(data: any): void {
-    this.websocketStream.send(data);
+    if (this.websocketStream) {
```

Comment:
Do we have a proper message if `this.websocketStream=false` ?

Reply:
No, we do not have a proper message. It is the same what we have  for  disconnect
```
  disconnect(): void {
    if (this.websocketStream) {
      this.websocketStream.close();
    }
  }
```
https://github.com/eclipse/che/blob/8b4076232a1450c674678a8dd2c94dc59d874a99/dashboard/src/components/api/json-rpc/websocket-client.ts#L88

I will add logs for this class as a separate commit on Monday. This code should be improved for adding a log message

Reply:
done